### ISSUE TITLE
installation command should be `rails spina:install` instead of `rails g spina:install`

### ISSUE THREAD

**Issue by shubharthaksangharsha** [SpinaCMS/Spina#1065]
Hey I've been trying to install spina on my rails project. I was reading the Github Readme. I added **gem "spina"** and do **bundle install** .Afterwards, I tried **rails g spina:install** as per the Readme. but it was throwing me an error regarding Mobility. 

```
/home/shubharthak/.rvm/gems/ruby-3.1.2/gems/mobility-1.2.9/lib/mobility.rb:133:in `translations_class': Mobility has not been configured. Configure with Mobility.configure, or assign a translations class with Mobility.translates_with(<class>) (Mobility::Error)
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/mobility-1.2.9/lib/mobility.rb:98:in `translates'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/spina-2.10.0/app/models/spina/resource.rb:9:in `<class:Resource>'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/spina-2.10.0/app/models/spina/resource.rb:2:in `<module:Spina>'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/spina-2.10.0/app/models/spina/resource.rb:1:in `<main>'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/kernel.rb:27:in `require'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:95:in `const_get'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:95:in `cget'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:237:in `block (2 levels) in eager_load'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:26:in `block in ls'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:18:in `each_child'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:18:in `ls'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:232:in `block in eager_load'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:217:in `synchronize'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:217:in `eager_load'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:317:in `each'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:317:in `eager_load_all'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/application/finisher.rb:74:in `block in <module:Finisher>'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/initializable.rb:32:in `instance_exec'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/initializable.rb:32:in `run'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/initializable.rb:61:in `block in run_initializers'
	from /home/shubharthak/.rvm/rubies/ruby-3.1.2/lib/ruby/3.1.0/tsort.rb:228:in `block in tsort_each'
	from /home/shubharthak/.rvm/rubies/ruby-3.1.2/lib/ruby/3.1.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
	from /home/shubharthak/.rvm/rubies/ruby-3.1.2/lib/ruby/3.1.0/tsort.rb:431:in `each_strongly_connected_component_from'
	from /home/shubharthak/.rvm/rubies/ruby-3.1.2/lib/ruby/3.1.0/tsort.rb:349:in `block in each_strongly_connected_component'
	from /home/shubharthak/.rvm/rubies/ruby-3.1.2/lib/ruby/3.1.0/tsort.rb:347:in `each'
	from /home/shubharthak/.rvm/rubies/ruby-3.1.2/lib/ruby/3.1.0/tsort.rb:347:in `call'
	from /home/shubharthak/.rvm/rubies/ruby-3.1.2/lib/ruby/3.1.0/tsort.rb:347:in `each_strongly_connected_component'
	from /home/shubharthak/.rvm/rubies/ruby-3.1.2/lib/ruby/3.1.0/tsort.rb:226:in `tsort_each'
	from /home/shubharthak/.rvm/rubies/ruby-3.1.2/lib/ruby/3.1.0/tsort.rb:205:in `tsort_each'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/initializable.rb:60:in `run_initializers'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/application.rb:372:in `initialize!'
	from /home/shubharthak/Desktop/curve_tomorrow/rails-base/config/environment.rb:5:in `<main>'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/zeitwerk-2.5.4/lib/zeitwerk/kernel.rb:35:in `require'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/application.rb:348:in `require_environment!'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/command/actions.rb:28:in `require_environment!'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/command/actions.rb:15:in `require_application_and_environment!'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/commands/generate/generate_command.rb:21:in `perform'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/command/base.rb:87:in `perform'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/command.rb:48:in `invoke'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/railties-7.0.3/lib/rails/commands.rb:18:in `<main>'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /home/shubharthak/.rvm/gems/ruby-3.1.2/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from bin/rails:4:in `<main>'
```
So, I suggest you to fix the Readme or atleast edit the Readme to add **rails spina:install** for the existing project.

**Comment by mattpolito** [comment]
@shubharthaksangharsha This seems like an awesome quick PR you could submit.

**Comment by Bramjetten** [comment]
That sounds like a peculiar bug, `rails spina:install` is just a rake task invoking `rails g spina:install`. It's a shorthand.

**Comment by shubharthaksangharsha** [comment]
Thank you @mattpolito I will definitely gonna do that

**Comment by shubharthaksangharsha** [comment]
@Bramjetten though it's a small bug, still took hours for me to debug it LOL. `rails g spina:install` actually wasn't working for me at all, maybe cuz I am working on an existing project but still I think it's a bug

**Comment by Bramjetten** [comment]
Changed instructions in readme.

**Comment by sedubois** [comment]
@Bramjetten I encounter the same Mobility error as in the OP even when using `rails "spina:install"` (*) after having defined a custom theme (**). The error is not there if I specify `default` or `demo` when asked which theme to use.

It looks like the error is raised at line 56 of `Spina::InstallGenerator`: `account.update(theme: theme)`. It looks like 

How can we install Spina using custom theme files? Using Spina 2.15.1.

(*) I actually used `rails_command "spina:install"` in a [custom Rails generator](https://edgeguides.rubyonrails.org/generators.html).
(**) I defined a custom theme by creating files `config/initializers/themes/my_theme.rb`, `app/views/layouts/my_theme/application.html.erb` and `app/views/my_theme/pages/*.html.erb` (as needed for the `view_templates` config). I did not find any documentation about this so this is based on reading the code in `Spina::InstallGenerator`. For the purpose of troubleshooting this issue, I just copied the contents of the `default` theme files.

NB: the Mobility error is actually present whether `rails g "spina:install"` or `rails "spina:install"`, it's just that in the latter case it's much less visible because only the error message is printed and not the stack trace.
